### DISCUSSION TITLE
Support preview feature in Landmark search views/ other search views.

### DISCUSSION THREAD

**Discussion by Quorafind** [tadashi-aikawa/obsidian-another-quick-switcher#108] [Ideas]
RT, now we have preview feature in GREP search view. If it possible to preview note in other search views?

![image](https://user-images.githubusercontent.com/13215013/192184508-6af3fa20-676c-4ce1-93e3-51105abe2b62.png)

**Comment by tadashi-aikawa** [comment]
@Quorafind 
No, there are not as far. But it would be possible. Could you create an FR issue?

**Reply by Quorafind** [reply]
Sure
